accepted/2022/version-selection.md

```diff
@@ -305,24 +323,27 @@ The SDK has similar concepts (per an [earlier design](https://github.com/dotnet/
 
 That takes care of the roll-forward ENVs. Let's look at what the CLI offers for the runtime:
 
-- `--roll-forward` -- specifies roll-forward policy
+- `--roll-forward` -- specifies runtime roll-forward policy
 
 We can follow the same pattern again, by offering a mirror SDK variant:
 
--- `--sdk-roll-forward`
+-- `--sdk-roll-forward` (default value is `major`)
 
-It is reasonable to ask the question of whether we could simply use the existing runtime-oriented ENVs and CLI switches to also affect the SDK. There are multiple scenarios where users would want separate handling (like with `dotnet test` or `dotnet run`):
+Note: The CLI doesn't offer a CLI affordance that matches `DOTNET_ROLL_FORWARD_TO_PRERELEASE`. We should wait for more feedback before adding pre-release-oriented switches to the CLI, for the runtime or SDK.
 
-- Roll-forward the SDK due to `global.json` pinning but retain runtime targeting as-is (as tested and expected).
-- Leave the SDK as-is (assume it works), but roll-forward the runtime from an end-of-life .NET version to a supported one.
+### SDK version selection
 
-It is also reasonable to ask why we need `--roll-forward` switches for the CLI at all (for the SDK or the runtime). It is useful to specify a minimum version and roll forward policy (via the CLI or otherwise) when you want to enable your code to run in environments you don't control. This could be equally as true for scripts as apps.
+The host offers `--fx-version` to select a specific runtime. We will add support to the same thing for selecting an SDK with `--sdk-version`.
 
-Note: The CLI doesn't offer a CLI affordance that matches `DOTNET_ROLL_FORWARD_TO_PRERELEASE`. We should wait for more feedback before adding pre-release-oriented switches to the CLI, for the runtime or SDK.
+### Ignore SDK version policies -- `--use-this`
 
-### Host SDK selection
+The SDK has version policies -- including `global.json` -- that you might just want to ignore. The following gesture is intended as the ultimate override. It will ignore `global.json` and any other SDK policies (ENVs or CLI params).
 
-The host offers `--fx-version` to select a specific runtime. We will add support to the same thing for selecting an SDK with `--sdk-version`.
+```bash
+dotnet run --use-this
+```
+
+Whatever `dotnet` you are using to run the command, you will get to use it. The `latestMajor` roll-forward policy will be applied.
```

Comment:
If we change the default for rollforward in global.json, here you could describe it as "The `dotnet` you're using to run the comment will be used and any `global.json` files will be ignored" (which today means `latestMajor` as well, so you can write it even today).

Reply:
Updated the text.